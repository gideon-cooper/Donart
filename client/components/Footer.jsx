import React from 'react'
import { Link } from 'react-router-dom'

export default function Footer () {
  return (
    <div className='footer'>
      <Link to={'/About'}>
        <div className='firstColumn'>
          <h1>About us</h1>
        </div>
      </Link>
      <div className='secondColumn'>
        <div>
          <h1>Contact us</h1>
          <br></br>
        </div>
        <div>
          <h1 className="gideonsLineBreaks">
            Copyright © 2020 – Donart® All Rights Reserved
            <br></br>
            Terms of Use | Privacy Policy | Disclaimer
          </h1>
        </div>
      </div>
      <div className='thirdColumn'>
        <h1>Legal</h1>
      </div>
    </div>
  )
}
