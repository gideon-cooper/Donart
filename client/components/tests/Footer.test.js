import React from 'react'
import { HashRouter as Router } from 'react-router-dom'
import renderer from 'react-test-renderer'

import Footer from '../Footer'

it('renders correctly', () => {
  const tree = renderer
    .create(
      <Router>
        <Footer />
      </Router>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
