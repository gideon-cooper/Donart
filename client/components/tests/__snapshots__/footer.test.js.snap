// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="footer"
>
  <a
    href="#/About"
    onClick={[Function]}
  >
    <div
      className="firstColumn"
    >
      <h1>
        About us
      </h1>
    </div>
  </a>
  <div
    className="secondColumn"
  >
    <div>
      <h1>
        Contact us
      </h1>
      <br />
    </div>
    <div>
      <h1
        className="gideonsLineBreaks"
      >
        Copyright © 2020 – Donart® All Rights Reserved
        <br />
        Terms of Use | Privacy Policy | Disclaimer
      </h1>
    </div>
  </div>
  <div
    className="thirdColumn"
  >
    <h1>
      Legal
    </h1>
  </div>
</div>
`;
